config/vufind/config.ini

```
@@ -2225,6 +2225,11 @@ includeSchemaOrgMetadata = true
 ; If set to "numerical", it will sort numerically by MARC tag.
 ;marcSubjectHeadingsSort = "numerical"
 
+; This setting (disabled by default) can be enabled to activate the Get This feature,
+; which opens a lightbox containing information on how to obtain records.
+; Feature related config are available in GetThis.yaml
```

Minor style suggestions:

```suggestion
; Feature-related configuration is available in GetThis.yaml
```